common/persistence/visibility/store/elasticsearch/query_interceptors_test.go

```diff
@@ -161,7 +161,7 @@ func (s *QueryInterceptorSuite) TestDurationProcessFunc() {
 		value     interface{}
 		returnErr bool
 	}{
-		{value: nil, returnErr: true},
+		{value: int64(1), returnErr: false},
 		{value: int64(1), returnErr: false},
```

Comment:
Two times, just to make sure :-) ?

Reply:
They are different test cases: one of them test plain integer, and the other is quoted.